packages/design-system-twrnc-preset/src/index.ts

```diff
@@ -4,3 +4,6 @@ export { Theme } from './Theme.types';
 
 // Hooks
 export { useTailwind, useTheme } from './hooks';
+
+// Config generation
+export { generateTailwindConfig } from './tailwind.config';
```

Comment:
We are exporting `generateTailwindConfig` so we can use it in the `tailwind-intellisense.config.js` file